<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Text Reader V1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="txt_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>691</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Apply</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_entry">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome to the text reader V1!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chng_lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono SemiLight</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter some text down below!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="link_lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change text first to reveal the link!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
